### DISCUSSION TITLE
YouTube 評論區翻譯

### DISCUSSION THREAD

**Discussion by N0stR3x** [immersive-translate/immersive-translate#1463] [Ideas]
### 新功能 | New Feature

1.YouTube 評論區翻譯

### 为什么您想要添加这些功能 | Why you want to add these features

1.作爲yt用戶 還是很建議加入這個功能 方便用戶觀看來自全世界網友們的評論

**Comment by keyboard3** [comment]
现在就能翻译啊

**Reply by N0stR3x** [reply]
只是想單翻譯評論區 沒有寫清楚 sry（
